packages/backend/src/helpers/setupPassportStrategies.js

```diff
@@ -0,0 +1,21 @@
+const RedditStrategy = require('passport-reddit').Strategy;
+
+const setupPassportStrategies = passport => {
+  passport.use(
+    new RedditStrategy(
+      {
+        clientID: 'rWjd7fOjL7eq3w',
```

Comment:
Are these id / secret host bound? 
If so, we can actually keep it as part of our devenv